srcd/contrib/docker/docker-compose.override.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.2'
+services:
+  sourced-ui:
+    user: 0:0
```

Comment:
what does it do?

Reply:
It runs the container as `root`, instead of as `superset` (the user **inside**, not the host one)

The reasoning is detailed in the PR description, section: **_"More info"_** (it is also added a link to an issue in Superset repo)

Reply:
YAML supports comments, let's add __short__ description there. Something like `set user as root inside the container to allow writing on host filesystem`

Reply:
well... that's not (fully) true, at least not directly. The whole sentence should be something like:
>_Set user as `root` inside the container to change `superset` internal `UID` to be the same as the host user. It will provide write access to the data from the volumes inside and outside of the container._

The problem is that the `user: 0:0` is not enough to _"allow writing on host filesystem"_.
The `user: 0:0` is (only) needed to change internal `superset` UID, and then run the `/entrypoint.sh` as `superset`, which will do provide the write access.

But if you feel comfortable with that other doc as you proposed, let's do it.

Reply:
docs added by d3dbe90292e4b14bf5569fe176849a0bf6edca86